buildspecs/build-nodeadm.yml

```diff
@@ -8,6 +8,9 @@ phases:
     - aws s3 cp _bin/arm64/nodeadm s3://$ARTIFACTS_BUCKET/latest-pre/linux/arm64/nodeadm
     - aws s3 cp _bin/e2e-test s3://$ARTIFACTS_BUCKET/latest-pre/e2e-test
     - aws s3 cp _bin/e2e.test s3://$ARTIFACTS_BUCKET/latest-pre/e2e.test
+    - mkdir -p /root/.docker && cp hack/docker-ecr-config.json /root/.docker/config.json
+    - ./test/e2e/cni/testdata/cilium/mirror-cilium.sh
+    - ./test/e2e/cni/testdata/calico/mirror-calico.sh 
```

Comment:
What would you think about putting this in a different stage? It could even run in parallel to the build one since there is no dependency. Although the main benefit to me is separation of concerns (pre-populating the cache is not part of building the binaries)